ui/push-health/details/DetailsPanel.jsx

```diff
@@ -169,15 +169,24 @@ class DetailsPanel extends React.Component {
               <TabList className="pl-0 w-100 list-inline">
                 <span className="d-flex justify-content-between w-100">
                   <span>
-                    <Tab className="font-weight-bold text-secondary list-inline-item pointable">
+                    <Tab className="font-weight-bold text-secondary list-inline-item">
                       Failure Summary
                     </Tab>
                     <Tab className="ml-3 font-weight-bold text-secondary list-inline-item pointable">
                       Log Viewer
                     </Tab>
-                    <Tab className="ml-3 font-weight-bold text-secondary list-inline-item pointable">
-                      Artifacts
-                    </Tab>
+                    {taskDetails.length > 0 ? (
+                      <Tab className="ml-3 font-weight-bold text-secondary list-inline-item pointable">
+                        Artifacts
+                      </Tab>
+                    ) : (
+                      <Tab
+                        className="ml-3 font-weight-bold list-inline-item pointable"
+                        disabled
+                      >
+                        Artifacts
```

Comment:
I wonder if we should hide this altogether or state that artifacts weren't found, so someone doesn't go back to treeherder or taskcluster to try to look for them?

Reply:
Good idea.